front/app/modules/commercial/moderation/admin/containers/SelectProject.tsx

```diff
@@ -1,71 +1,66 @@
 import React, { memo, useCallback } from 'react';
-import { adopt } from 'react-adopt';
+
+// api
+import useProjects from 'api/projects/useProjects';
+
+// components
 import FilterSelector from 'components/FilterSelector';
 import useLocalize from 'hooks/useLocalize';
 
 // i18n
 import { FormattedMessage } from 'utils/cl-intl';
 import messages from './messages';
 
-// resources
-import GetProjects, { GetProjectsChildProps } from 'resources/GetProjects';
+// utils
 import { isNilOrError } from 'utils/helperUtils';
 
-interface InputProps {
+// typings
+import { PublicationStatus } from 'api/projects/types';
+
+interface Props {
   onChange: (projectIds: string[]) => void;
   selectedProjectIds: string[];
 }
-interface DataProps {
-  projects: GetProjectsChildProps;
-}
 
-interface Props extends DataProps, InputProps {}
+const PUBLICATION_STATUSES: PublicationStatus[] = [
+  'published',
+  'archived',
+  'draft',
+];
 
-const SelectProject = memo(
-  ({ onChange, projects, selectedProjectIds }: Props) => {
-    const localize = useLocalize();
-    const handleOnChange = useCallback((newProjectIds: string[]) => {
-      onChange(newProjectIds);
-      // eslint-disable-next-line react-hooks/exhaustive-deps
-    }, []);
+const SelectProject = memo(({ onChange, selectedProjectIds }: Props) => {
+  const localize = useLocalize();
+  const { data: projects } = useProjects({
+    publicationStatuses: PUBLICATION_STATUSES,
+    canModerate: true,
+  });
 
-    if (!isNilOrError(projects) && projects.length > 0) {
-      const values = projects.map((project) => {
-        return {
-          text: localize(project.attributes.title_multiloc),
-          value: project.id,
-        };
-      });
+  const handleOnChange = useCallback((newProjectIds: string[]) => {
+    onChange(newProjectIds);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Can we remove this - we should follow the rule, generally.

Reply:
It was already like this, and I don't like removing it because perhaps there's a good reason for it (`FilterSelector` might need a stable reference, for example)